test-apps/display-test-app/src/frontend/ViewAttributes.ts

```diff
@@ -471,6 +487,11 @@ export class ViewAttributes {
     this._updates.push((view) => thematic.update(view));
   }
 
+  private addAtmosphere(): void {
+    const atmosphere = new AtmosphereEditor(this._vp, this._element);
+    this._updates.push((view) => atmosphere.update(view));
+  }
+
   private getBackgroundMap(view: ViewState) { return view.displayStyle.settings.backgroundMap; }
   private addBackgroundMapOrTerrain(): void {
     const isMapSupported = (view: ViewState) => view.is3d() && view.iModel.isGeoLocated;
```

Comment:
add + run Image Tests

Reply:
This change has been reverted

Reply:
Still should run Image Tests to verify all is good

Reply:
we went over these together and they looked good.